server/test/controllers/admin/AdminProgramBlockPredicatesControllerTest.java

```diff
@@ -39,12 +40,11 @@ public void setup() {
   }
 
   @Test
-  public void edit_withInvalidProgram_notFound() {
-    Http.Request request = fakeRequest().build();
-
-    Result result = controller.edit(request, 1L, 1L);
-
-    assertThat(result.status()).isEqualTo(NOT_FOUND);
+  public void edit_withNonExistantProgram_notFound() {
```

Comment:
This test body is the same as edit_withActiveProgram_throws below. I think it needs a different body to be consistent with it's current name.

Reply:
done!